frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import messageIcon from '../assets/messageIcon.svg'
+import adminIcon from '../assets/adminIcon.svg'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
```

Comment:
move this 3 lines up, and combine line 6 and 8

Reply:
Resolved in commit 6c4688c5a5c1853aa8ff09a0d49fdb749f87bdbb